packages/esm-ward-app/src/config-schema.ts

```diff
@@ -207,23 +149,33 @@ export interface WardConfigObject {
 }
 
 export interface WardPatientCardsConfig {
-  patientCardElementDefinitions: Array<PatientCardElementDefinition>;
+  obsElementDefinitions: Array<ObsElementDefinition>;
+  identifierElementDefinitions: Array<IdentifierElementDefinition>;
```

Comment:
I think `patientAddressElementFields` should belong here instead of the `WardPatientCardDefinition` interface.

Reply:
Sounds good. That does mean that the same address elements will necessarily be used for every ward, but that's probably fine.

Reply:
Wait... why? Each value in `obsElementDefinitions` and `identifierElementDefinitions` can specify which element id to apply the config to. Can't we just do the same thing with `patientAddressElementFields`?

Reply:
Oh I see, is what you're suggesting that instead of having 'patient-address' be a built-in element that is configurable, that instead we should provide a system for defining elements of type 'patient-address' which can then be included, but where the only real difference between the different patient-address elements is which fields are shown?

Reply:
Yeah. That follows the same pattern as the `identiferElement` and `obsElement`, right?

Reply:
Sure, it does.